### PR TITLE
fix: Update dualtoken.py

### PR DESCRIPTION
**Pull request by msampathkumar**
Minor Update.

Using `base64_encoder` instead of `urlsafe_b64encode`

## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by justin-mp — APPROVED** [review]

**Review by rsamborski — DISMISSED** [review]
I have just noticed that `mediacdn_dualtoken_sign_path_component` region tag doesn't include any imports. Please make sure it's added it in the top (around line 17) to include imports used in that sample (base64, ed25519, datetime).

**Comment by msampathkumar** [comment]
Update the PR. Included the imports in the region tags.

File: media_cdn/dualtoken.py [hunk 1]
```diff
@@ -209,10 +210,9 @@ def sign_path_component(
         key_name=key_name,
     )
 
-    digest = ed25519.Ed25519PrivateKey.from_private_bytes(decoded_key).sign(
-        policy.encode("utf-8")
-    )
-    signature = base64.urlsafe_b64encode(digest).decode("utf-8")
+    digest = ed25519.Ed25519PrivateKey.from_private_bytes(
+        decoded_key).sign(policy.encode('utf-8'))
+    signature = base64.base64_encoder(digest).decode('utf-8')
```
Comment (dandhlee) [inline media_cdn/dualtoken.py:215]: `AttributeError: module 'base64' has no attribute 'base64_encoder'`

File: media_cdn/dualtoken.py [hunk 2]
```diff
@@ -15,14 +15,15 @@
...
+# [START mediacdn_dualtoken_sign_path_component]
 import base64
 import datetime
 import hashlib
 import hmac
 
 import cryptography.hazmat.primitives.asymmetric.ed25519 as ed25519
 
-
+# [END mediacdn_dualtoken_sign_path_component]
```
Comment (dandhlee) [inline media_cdn/dualtoken.py:26]: Can this be right below the import without a space? It will also help solve the lint issue.

**Review by dandhlee — CHANGES_REQUESTED** [review]
Two pending failures (lint & test)

**Comment by msampathkumar** [comment]
Resolved the lint issues also updated the test file accordingly.

**Review by m-strzelczyk — APPROVED** [review]

**Review by dandhlee — APPROVED** [review]

**Comment by dandhlee** [comment]
Dismissing @rsamborski's stale review, please feel free to re-raise it but looks good to me now.

**Review by rsamborski — APPROVED** [review]
Sorry for not looking at this for a while due to PTO. I am approving and merging it right now.

**Pull request merged**